README.md

```diff
@@ -20,9 +20,10 @@ e.g. `ac.can(role).execute('create').on(resource)`
 - Grant permissions by attributes defined by glob notation (with nested object support).
 - Ability to filter data (model) instance by allowed attributes.
 - Ability to control access using conditions.
-- Supports AND, OR, NOT, EQUALS, NOT_EQUALS, STARTS_WITH, LIST_CONTAINS conditions.
-- You can specify dynamic context values in the conditions using JSON Paths.
-- You can define your own condition functions too but please note if you use custom functions instead of standard conditions, you won't be able to save them as json in the DB.
+- Supports AND, OR, NOT, EQUALS, NOT_EQUALS, STARTS_WITH, LIST_CONTAINS core conditions.
+- You can specify dynamic context values in the core conditions using JSON Paths.
+- Supports your own custom conditions e.g. `custom:isArticleOwner`.
+- You can define your own function conditions too but please note if you use custom functions instead of standard conditions, you won't be able to save them as json in the DB.
```

Comment:
This is no longer true, we are supporting storing of custom functions

Reply:
This is what the old README says. Can you elaborate on this please? How a pure JS function is serialized?

Reply:
https://github.com/tensult/role-acl/blob/develop/src/utils/common.ts#L36
Here is the small function I am using for the same.

Reply:
Good to know! Not my "style" but it's a neat feature :)